api/internal/impl/api/health/get.go

```diff
@@ -0,0 +1,45 @@
+package health
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/spiffe/go-spiffe/v2/workloadapi"
+	"github.com/spiffe/spike-sdk-go/api/entity/data"
+	"github.com/spiffe/spike-sdk-go/api/entity/v1/reqres"
+	"github.com/spiffe/spike-sdk-go/api/url"
+	"github.com/spiffe/spike-sdk-go/net"
+)
+
+// GetSystemStatus fetches the current system status from the operator endpoint using POST
+func GetSystemStatus(ctx context.Context, source *workloadapi.X509Source) (*data.StatusResponse, error) {
+
+	client, err := net.CreateMTLSClient(source)
```

Comment:
this might have changed in `main`; merge with main to make sure you won't get merge conflicts later.